pkg/odo/cli/service/create.go

```diff
@@ -36,6 +39,10 @@ var (
     # Create new postgresql service from service catalog using dev plan and name my-postgresql-db.
     %[1]s dh-postgresql-apb my-postgresql-db --plan dev -p postgresql_user=luke -p postgresql_password=secret`)
 
+	createOperatorExample = ktemplates.Examples(`
+	# Create new EtcdCluster service from etcdoperator.v0.9.4 operator.
```

Comment:
I'm not denying your point. But we're commenting this because the CRDs provided by `etcdoperator.v.0.9.4` operator are `EtcdCluster`, `EtcdBackup` and `EtcdRestore`. So, we're just using the name of CRDs as-is.